dask/dataframe/io/orc.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import absolute_import, division, print_function
+
+from ..core import DataFrame
+from ...base import tokenize
+from ...bytes.core import get_fs_token_paths
+from ...utils import import_required
+from .parquet import _get_pyarrow_dtypes, _meta_from_dtypes
+from ...compatibility import string_types
+
+__all__ = ('read_orc',)
+
+
+def _read_orc_stripe(fs, path, stripe, columns=None):
+    """Pull out specific data from specific part of ORC file"""
+    orc = import_required('pyarrow.orc', 'Please install dev version pyarrow')
+    with fs.open(path, 'rb') as f:
```

Comment:
Using just `OpenFile`s from `open_files`, got some crashes here, seems that one thread closes the file another is working on - perhaps within the orc code?

Reply:
There were some crashes due to an arrow bug that I fixed a while ago, not sure if the version you're using had that. If you can provide a reproducible test case I can take a look at some point.